Cargo.toml

```diff
@@ -11,16 +11,26 @@ documentation = "https://docs.rs/raft"
 description = "The rust language implementation of Raft algorithm."
 categories = ["algorithms", "database-implementations"]
 edition = "2018"
+build = "build.rs"
+
+[build-dependencies]
+protoc-rust = "~2.0-2.2"
+regex = "1.1"
+protobuf-build = { git = "https://github.com/nrc/kvproto-build" }
 
 [features]
 default = []
+regenerate = []
```

Comment:
Could we use `gen` here? (And `prost-codec`/`protobuf-codec` for the codec libs). I'm trying to be consistent on these options across all the crates.